.github/workflows/build-ami.yml

```diff
@@ -0,0 +1,50 @@
+name: Build AWS Neuron AMI
+
+on:
+    push:
+      branches:
+        - main
+      paths:
+        - 'aws-ami-creation/**'
+  workflow_dispatch:
```

Comment:
We should be able to pass which "tag" to build. That on a cron schedule it builds the main branch but that we can use the same action for building a release image we can then publish.